README.md

```diff
@@ -5,10 +5,8 @@
 
 # ⚡️ "CIRCT" / Circuit IR Compilers and Tools
 
-"CIRCT" stands for "Circuit IR Compilers and Tools".  One might also interpret
```

Comment:
I personally am a fan of the cheeky intro, but I think that explicitly mentioning MLIR in the first paragraph is a good idea!